app_template_iframe/assets/iframe.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+  <p>It works!</p>
```

Comment:
potentially some description of what works (the app, sdk, etc.) how it works and where to get more information of developer resources?